react/components/Dropdown/index.js

```diff
@@ -180,7 +180,7 @@ class Dropdown extends Component {
         <label>
           {label && <span className={labelClasses}>{label}</span>}
           <div className={containerClasses} style={containerStyle}>
-            <div id={id} className={`vtex-dropdown__button ${classes}`}>
+            <div className={`vtex-dropdown__button ${classes}`}>
```

Comment:
Revert `id` and add `testid` to the select element. Will be a minor instead of a breaking change.